srcpkgs/python3-dbus-next/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'python3-dbus-next'
+pkgname=python3-dbus-next
+version=0.2.3
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3"
+short_desc="Next great DBus library for Python"
```

Comment:
```suggestion
short_desc="Next great DBus library"
```
self-evident from the package name

Reply:
I've taken it directly from upstream's README.md ;-) Fixed.